Formula/g/gradle.rb

```diff
@@ -15,6 +15,7 @@ class Gradle < Formula
   end
 
   # https://github.com/gradle/gradle/blob/master/platforms/documentation/docs/src/docs/userguide/releases/compatibility.adoc
+  depends_on "gradle-completion"
```

Comment:
```suggestion
  depends_on "gradle-completion"
  # https://github.com/gradle/gradle/blob/master/platforms/documentation/docs/src/docs/userguide/releases/compatibility.adoc
```
We should probably keep the comment where it is.